### ISSUE TITLE
Content bug: Page for ARIA role spinbutton not found

### ISSUE THREAD

**Issue by novellac** [mdn/content#13421]
## What page(s) did you find the problem on?

Spinbutton role page (leads to 404): https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/spinbutton_role
(linked from [Composite role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/composite_role))

## Specific page section or heading?

N/A

## What is the problem?

It looks like the folder and index.md for the spinbutton_role entry is missing.

## What did you expect to see?

- I expected the link for the ARIA [spinbutton role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/spinbutton_role) to lead to a page describing the use of the spinbutton role
- I also expected an `index.md` file to exist on the path `files/en-us/web/accessibility/aria/roles/spinbutton_role`, because other roles such as `tablist_role` do exist here.

## Did you test this? If so, how?

- I clicked the link and found that spinbutton_role has a 404 in the network tab
- I forked the project and saw that no folder and index.md file exists for `spinbutton_role` on the `files/en-us/web/accessibility/aria/roles` path (where I'd expect to find it).

---------

Suggested fix: Add a `spinbutton_role/index.md` with the requisite info at `files/en-us/web/accessibility/aria/roles`

**Comment by estelle** [comment]
https://github.com/mdn/content/pull/14764

**Comment by estelle** [comment]
https://github.com/mdn/content/pull/15491
